q2cli/commands.py

```diff
@@ -184,7 +186,7 @@ def __call__(self, **kwargs):
         import os
         import qiime2.util
 
-        arguments, missing_in, verbose = self.handle_in_params(kwargs)
+        arguments, missing_in, verbose, quiet = self.handle_in_params(kwargs)
```

Comment:
Raise an exception if quiet and verbose are both true. (We're planning to make these mutually exclusive)